app/uk/gov/hmrc/gform/formtemplate/Rewriter.scala

```diff
@@ -369,17 +377,29 @@ trait Rewriter {
         section match {
           case s: Section.NonRepeatingPage =>
             s.copy(
-              page = s.page.copy(includeIf = replaceIncludeIf(s.page.includeIf), fields = replaceFields(s.page.fields))
+              page = s.page.copy(
+                includeIf = replaceIncludeIf(s.page.includeIf),
+                fields = replaceFields(s.page.fields),
+                redirects = s.page.redirects.map(_.map(r => r.copy(ifExpr = replaceIncludeIf(r.ifExpr))))
```

Comment:
Can we introduce `replaceRedirects` method and reuse it?

This
```
page.redirects.map(_.map(r => r.copy(ifExpr = replaceIncludeIf(r.ifExpr))))
```
Is repeated 3x here 👎